Documentation/helm-operator.md

```diff
@@ -107,13 +107,18 @@ The following tables lists the configurable parameters of the rook-operator char
 | `agent.mountSecurityMode`    | Mount Security Mode for the agent.                                                                      | `Any`                                                  |
 | `agent.toleration`           | Toleration for the agent pods                                                                           | <none>                                                 |
 | `agent.tolerationKey`        | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `agent.nodeAffinity`         | The node labels for affinity of `rook-agent`                                                            | <none>                                                 |
 | `discover.toleration`        | Toleration for the discover pods                                                                        | <none>                                                 |
 | `discover.tolerationKey`     | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `discover.nodeAffinity`      | The node labels for affinity of `discover-agent`                                                        | <none>                                                 |
```

Comment:
Could you add the three `*` in brackets as on line 105 to your entry, and the one for `agent.mounts` with two `*` which is missing it.

Reply:
done.